pkg/metrics/controller_test.go

```diff
@@ -292,7 +291,6 @@ func TestControllerGameServersTotal(t *testing.T) {
 func TestControllerFleetReplicasCount(t *testing.T) {
 	runtime.FeatureTestMutex.Lock()
 	defer runtime.FeatureTestMutex.Unlock()
-	require.NoError(t, runtime.ParseFeatures(fmt.Sprintf("%s=false", runtime.FeatureResetMetricsOnDelete)))
```

Comment:
Since this test was only going to pass when the false was false, the entire test can be deleted.

Reply:
I had the same thought, but I was uncertain about updating the 'controller.go' file.